pal/pacer/pacer-web.ts

```diff
@@ -25,20 +25,22 @@
 import { EDITOR } from 'internal:constants';
 import { assertIsTrue } from '../../cocos/core/data/utils/asserts';
 
+const FRAME_RESET_TIME = 2000;
```

Comment:
What does this variable mean?

Reply:
2000 represents do reset each 2 seconds, that is, a reset is performed every 120 ticks on a 60hz screen. I'm not sure whether 2000 is the most suitable value. However, there is no problem with running, so it may be an appropriate value